.pre-commit-config.yaml

```diff
@@ -3,7 +3,6 @@ repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v2.3.0
     hooks:
-    -   id: check-yaml
```

Comment:
Why was this removed?

Reply:
Because we are using yaml templates in meta.yaml, check-yaml isn't powerful enough to deal with these files. We should look into alternatives.